scripts/teamDataScript.js

```diff
@@ -0,0 +1,617 @@
+{
+    $('#teamDataAlertModal').hide();
+}
+
+function displayTeamRadioList()
+{
+    var season = getCookie('currCheckedSeason');
+    var event = getCurrEvent();
+    if (season.trim().length == 0 || event.trim().length == 0) return;
+
+    var teamsJSON = getTeams(season,event);
+    var keys = Object.keys(teamsJSON);
+    for (var i = 0; i < keys.length; i++)
+    {
+        var teamNumber = keys[i];
+        var teamName = teamsJSON[teamNumber];
+
+        var newItem = "<label><input type=\"radio\" name=\"teamListItem\" value=\"" + teamNumber + "\">" + `${teamNumber}: ${teamName}` + "</label><br>";
+        $("#teamRadioList").append(newItem); //add to radiolist
+    }
+}
+
+//onchange for the team radio list
+function changeCurrTeam()
+{
+    var currTeamNumber = $("input[name=teamListItem]:checked","#teamRadioList").val(); //in prev method, val of each radio item is the tNum
+    var currSeason = getCookie('currCheckedSeason');
+    var currEvent = getCurrEvent();
+    var cname = 'currCheckedTeam';
+    var currTeamName = getTeams(currSeason,currEvent)[currTeamNumber];
+
+    if (currSeason.trim().length == 0 || currEvent.trim().length==0 || currTeamNumber.trim().length == 0)
+    {
+        $("#teamStatsTitle").text('');
+        return;
+    }
+
+    var cvalue = `/${currSeason}/${currEvent}/${currTeamNumber}`;
+    setCookie(cname,cvalue,750);     //curr team cookie --> currCheckedTeam = /{season}/{event}/{currTeamNumber}
+
+    $("#teamStatsTitle").text(`Data for team ${currTeamNumber} : ${currTeamName}`);
+    $('#pitScoutingTitle').text(`Pit Scouting Questions and Answers for team ${currTeamNumber}: ${currTeamName}`);
+    $('.js_clear_on_load').val("").html("");
+
+    loadPit();
+    loadMatch();
+    localStorage.setItem('currCheckedMatch','');
+}
+
+//basically copy pasted from w3schools, onclick for a change in tabs
+function displayTabContent(evt, tabName)
+{
+    var i, x, tablinks;
+    x = document.getElementsByClassName("tabContent");
+    for (i = 0; i < x.length; i++) {
+      x[i].style.display = "none";
+    }
+    tablinks = document.getElementsByClassName("tabs");
+    for (i = 0; i < x.length; i++) {
+      tablinks[i].className = tablinks[i].className.replace(" w3-bottombar-gold", " w3-bottombar-maroon");
+    }
+    document.getElementById(tabName).style.display = "block";
+    evt.currentTarget.firstElementChild.className += " w3-bottombar-gold";
+
+    if (tabName.startsWith('pit')) //function calls for pit page
+    {
+        $('.js_clear_on_load').val('').html('');
+        loadPit();
+    } else //function calls for match page
+    {
+        loadMatch();
+    }
+    localStorage.setItem('currCheckedMatch','');
+}
+
+
+
+
+//Pit stuff
+function loadPit()
+{
+    var season = getCookie('currCheckedSeason');
+    var cname = season + ' pitQuestions';
+    var event = getCurrEvent();
+    var questions = getCookie(cname).split('Ω'); //Ω is separator for questions
```

Comment:
good that you recognize you need to clarify the 'Ω' with a comment.
Event better would be to store the character in a constant in a shared file (like cookie helpers) like:
```js
const COOKIE_STRING_SEPERATOR = 'Ω';
```
then when you re-use it here, you can refer to it by name and the comment becomes unneeded because the code itself shows why you're splitting on that.